pkg/menu/menu_test.go

```diff
@@ -0,0 +1,36 @@
+package menu
+
+import (
+    "strconv"
+    "testing"
+    ui "github.com/gizak/termui/v3"
+)
+
+func Test_newParagraph(t *testing.T){
+    if err := ui.Init(); err != nil {
+        t.Log(err)
+    }
+    defer ui.Close()
+    p := newParagraph("newParagraph test", false, 0, 50, 3)
+    t.Log(p.Text)
+}
```

Comment:
How do you know it worked? The test needs to be in this code. It seems you are look at p.Text for some indication. Can you compare p.Text to a string and see if it has an expected value?

Reply:
remember, when you run a test, it must be able to run as part of an unattended test, where you do not watch it run. It will be run by constant integration framework, for example. So your test has to have its own checks to make sure it worked. Just printing things via t.Log is not enough.